pom.xml

```diff
@@ -763,6 +763,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.1.1</version>
```

Comment:
A common version is already defined in `pluginManagement` https://github.com/codice/ddf/blob/master/pom.xml#L579-L583.

While you're in here you could remove the version definition from everywhere that defines a custom version if you wanted, but that's a ✏️

Reply:
ok